index.html

```diff
@@ -101,6 +101,88 @@ <h5>Skills</h5>
           >
         </div>
       </header>
+      <section>
+        <h1>MY RECENT WORK</h1>
+        <div class="seperator"></div>
+        <div class="cards">
+          <div class="card">
```

Comment:
- [ ] Good job introducing the works section with a `<section>` tag! However, as per requirement, the list of works is supposed to be positioned using the CSS property of `display:grid`, please make sure to change the CSS accordingly.

Reply:
Hi,
Thanks for review. But, I do not know what happened but locally it loads nearly equal to the design. take a look at the screen shot below

![image](https://user-images.githubusercontent.com/22321030/148557911-14b599f8-70c0-4ee5-8ed6-9383e8036722.png)

Could you help me trouble shoot why!

Reply:
the .cards class has a display: grid

Reply:
Hi @mnekx, are you sure you pushed those changes? 
For the current pull request, you're requesting to merge the branch `work_sect` into `main`. 
If I take a look at this branch's code, the website looks like in the screenshot that I showed you.
Maybe you made changes on another branch and forgot to push them to this one?
Take a look at the [files changed](https://github.com/mnekx/my-portfolio/pull/5/files) section of this pull request and you'll see that your CSS doesn't have any display:grid or display:flex in it.